extensions/wikia/ArticleVideoMVP/templates/ArticleVideoMVP_index.php

```diff
@@ -0,0 +1,17 @@
+<div id="premium-mvp-video" style="background-image: url(<?= $thumbnailUrl; ?>);">
+	<div class="video-container">
+		<div class="video-placeholder">
+			<div class="video-thumbnail" style="background-image: url(<?= $thumbnailUrl; ?>);">
+				<div class="close"></div>
```

Comment:
I'm always not sure about empty HTML markup. It makes me feel that we made something wrong. 

In this case we could use `img` tag that allows you to specify alternative tag when there is connection issue.